chains_sample.json

```diff
@@ -0,0 +1,329 @@
+[
```

Comment:
this sample file doesn't seem to be used anywhere, do we need it?

Reply:
Agreed. I've removed it. I originally had it there in case people want to test the script without running on the entire JSON file since it's quite large. But people can just create it on their own if they need it.